sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSessionManagerTest.java

```diff
@@ -289,197 +237,6 @@ void singleUnnamedSessionLockRenew() {
             .verify(Duration.ofSeconds(45));
 
         assertTrue(sessionLockRenewCalled.get());
-
-    }
-
-    /**
-     * Verify that when we receive multiple sessions, it'll change to the next session when one is complete.
-     */
-    @Test
-    void multipleSessions() {
```

Comment:
The v1 `ServiceBusSessionManager` is removed. The tests `multipleSessions` and `multipleReceiveUnnamedSession` are not valid for `ServiceBusSingleSessionManager` as it streams from one session only. By removing `ServiceBusSessionManager`, the session enabled `ServiceBusReceiverAsyncClient` is no longer responsible for backing processor, but `SessionsMessagePump` (added in v2).